app/code/Magento/GraphQl/Controller/GraphQl.php

```diff
@@ -115,6 +124,7 @@ class GraphQl implements FrontControllerInterface
      * @param ContextFactoryInterface $contextFactory
      * @param LogData $logDataHelper
      * @param LoggerPool $loggerPool
+     * @param AreaList $areaList
```

Comment:
@shikhamis11 can you add really quick the " | null" here to be consistent with the constructor?